### ISSUE TITLE
run some population focused models, maybe?

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/wildhellgarden#13]
@dbuona I am not sure if we need more models for this (as I am not sure how the current models work) but can we make some figures showing population variation WITHIN species for budset and leafout? Ideally use the current models to plot the population differences for each species and phenophase... I think these will be good for the supp and maybe a line in the main text about how consistent (or not) population effects were for different species.

**Comment by dbuona** [comment]
@lizzieinvancouver because the current variance partitioning models are "intercept only", the effects of population and species are additive so they all basically look the same [see here](https://github.com/lizzieinvancouver/wildhellgarden/blob/master/analyses/figures/SPxPOP.jpeg). I think we do need a different model if we want to do this.

**Comment by lizzieinvancouver** [comment]
@dbuona I think these plots do show population-level differences for each species, right? If you look at the output are there different estimates for each population x species? If so, I think the model is doing what I am interested in and we just need to check and be able to say something like (what I think I see, but I am glancing):

> Population-level differences were similar across species (see Table), with WM usually doing X and some other site at the other extreme. 

Rob Guy was just wondering if there were BIGGER population level effects in some species and it looks like the answer is no ....

**Comment by dbuona** [comment]
@lizzieinvancouver checkout[ this plot of raw data and mean+ se](https://github.com/lizzieinvancouver/wildhellgarden/blob/master/analyses/figures/rawpopulations.jpeg)
and [this plot of fitted values over data](https://github.com/lizzieinvancouver/wildhellgarden/blob/master/analyses/figures/fittedpopulations.jpeg)

I think the main criticism of inference from our model is that in the raw data, we often only have 2 or 3 sites per species so there is a lot of shrinkage going on in the fitted estiamtes. That said even in the raw data, there aren't really any significant differences between populations for most species.

**Comment by lizzieinvancouver** [comment]
@dbuona Interesting! Indeed there is a ton of shrinkage -- and I am impressed the model picks out what we know as a current version of understanding (from totally separate data) of biology: leafout does not have population structure but budset does. There's no evidence in the raw data of massive structure (though someone could certainly come up with stories, but they could come up with them for leafout too then) which makes me feel better. Also, one take-away I have here is that we should delete out showing estimates when we have zero data from the site. It will be fiddly (perhaps a good place to ask for co-author help) but we should make the supplemental figure only show the species x population estimates where we had data for that species x population, even though the model estimates the full ensemble.

Also, can you make these plots or something similar for year? I think one point we need to make sure is in the paper is that year effects on budset are larger than population (are they? I think they are but we probably need to say this as it flies in the face of most assumed biology about budset just now).

**Comment by dbuona** [comment]
I just pushed two new figures fittedpopulations.jpeg and fittedpopulations_x_year.jpeg. Let me know if that is what you had in mind.

**Comment by lizzieinvancouver** [comment]
@dbuona Holymoly is budset different in 2020! These figures are basically what I wanted, but it's hard to compare years in fittedpopulations_x_year -- can you make the years different symbols so you can smush the populations together or somehow make it easier to compare years?

**Comment by lizzieinvancouver** [comment]
@dbuona Looks good! It's now easy for me to eyeball and say, 'whoa! Budset is like 10 days later in 2020 (?) than the other years across populations!' which I am not sure most budset people would predict. I think we now just need to make sure we have well categorized the years so this caption in the supplement can say something about how the climate varied from 2018 to 2019 to what I think is 2020.

**Comment by dbuona** [comment]
@lizzieinvancouver is it possible this is a sampling effect?

"For the years of 2018-2019, we made phenological observations of all individuals in the common garden twice per week from February to December. In 2020 due to the COVID 19 pandemic, we monitored once per week from March to November."

We only monitored once a week in 2020 and twice per week in the other 2 years.

**Comment by lizzieinvancouver** [comment]
@dbuona Good point and seems worth checking! Though a priori it should not lead to a 10 day difference if the sampling days error effectively went from 3.5 to 7 days. I would plot our day of estimated budset versus the raw observations for a few individuals comparing the 3 years and see what we get. If we see a problem then we should think on what to do next. If you don't see a problem then I think we can be up front but also point out that we checked this and give the reader some comparisons, like 'though our sampling during 2020 due to the Covid 19 pandemic was less frequent (1X per week versus 2X per week) this difference in days is much smaller than observed differences across years (e.g., XX change for this species and YY change for that species).

This is also where checking the climate and leafout should give us inference. Is there a climatic or phenological reason that 2020 was later (while remembering leafout could also be affected, so if leafout is late we should see the spring as colder).
